### DISCUSSION TITLE
Volantex 768-1 protocol

### DISCUSSION THREAD

**Discussion by FuryIII** [pascallanger/DIY-Multiprotocol-TX-Module#844] [General]
Can anyone tell me if there's a protocol for Volantex's 750mm P51?  768-1.

**Comment by pascallanger** [comment]
Never seen this RX but if it's the same as this one it can't be emulated: https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/issues/412

**Comment by FuryIII** [comment]
Not sure if it's the same one, looks similar though. Shame it can't be emulated. Looks like I'll need to swap out the receiver to something my TX16s will talk to. 
 Thank you for the response though!
    On Sunday, April 9, 2023 at 09:30:53 AM EDT, pascallanger ***@***.***> wrote:  
 
 


Never seen this RX but if it's the same as this one it can't be emulated: #412
